examples/spark/spark-gluster/spark-master-controller.yaml

```diff
@@ -0,0 +1,34 @@
+kind: ReplicationController
+apiVersion: v1
+metadata:
+  name: spark-master-controller
+  labels:
+    component: spark-master-controller
+spec:
+  replicas: 1
+  selector:
+    component: spark-master
+  template:
+    metadata:
+      labels:
+        component: spark-master
+    spec:
+      containers:
+        - name: spark-master
+          image: gcr.io/google_containers/spark-master
+          securityContext:
+            privileged: true
```

Comment:
You're correct.  I've re-tested without privileges on the master w/o issue.  Will push it again with the fixed master controller.